netci.groovy

```diff
@@ -115,18 +115,18 @@ def CreateBuildTasks = { machine, configTag, buildExtra, testExtra, runCodeAnaly
 
 def CreateLinuxBuildTasks = { machine, configTag, linuxBranch, nonDefaultTaskSetup ->
     [true, false].each { isPR ->
-        ['debug', 'release'].each { buildType ->
+        ['debug', 'release', 'test-build'].each { buildType ->
```

Comment:
We previously called this configuration "test", but for Linux I think this makes sense since this simplifies the logic around selecting the buildFlag below.

Do we expect those flags to be stable into the future?

I meant to comment on this before but does it bother anyone else that the only build-flavor-related flag that has a `-build` suffix is `--test-build`? I understand that the disambiguation is important but considering that we've only got one flag to "run tests" wouldn't it make sense to call that "--run-tests" and keep the flags for build configurations simple?

cc @digitalinfinity @jianchun